test/controllers/countries_controller_test.rb

```diff
@@ -0,0 +1,49 @@
+require 'test_helper'
+
+class CountriesControllerTest < ActionController::TestCase
+  setup do
+    @country = countries(:one)
+  end
+
+  test "should get index" do
+    get :index
+    assert_response :success
+    assert_not_nil assigns(:countries)
+  end
+
+  test "should get new" do
+    get :new
+    assert_response :success
+  end
+
+  test "should create country" do
+    assert_difference('Country.count') do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.